src/module/actor/sheet/helpers.ts

```diff
@@ -80,4 +80,36 @@ function createAbilityViewData(item: AbilityItemPF2e | FeatPF2e): AbilityViewDat
     };
 }
 
-export { condenseSenses, createAbilityViewData, createBulkPerLabel, onClickCreateSpell };
+/**
+ * Applies a delta change to the value of an input element
+ *
+ * @param input reference to the input field
+ * @param delta the amount of change to apply to the input's current value
+ * @param [min] minimum input value
+ * @param [max] maximum input value; if "0", do not limit the value
+ * @param [triggerChange] if true, dispatches a change event after this operation is complete
+ * @returns the new value of the input field
+ */
+function applyDeltaToInput(
+    input: HTMLInputElement,
+    delta: number,
+    min: number = 0,
+    max: number = 0,
+    triggerChange: boolean = true,
+): string {
```

Comment:
Given you're passing the input directly, I think you can pull min/max from this function directly. You don't use the triggerChange param either so you can probably omit it.

Not step though, since you're passing a delta and its part of the delta computation.

Reply:
I can take care of it. With regards to step, you think I should leave the `delta` param in the helper but remove the `stepSize` bits from the base sheet, passing in `1` or `-1` directly?

Reply:
Exactly that.